Skills Consistency- Items Folder

rosebrallier opened the pull request:
Re Issue #372

rosebrallier committed:
Update 04_01_Melee_&_Thrown_Weapons.txt

Note: C10 Jury-rigging DC is completely arbitrary

rosebrallier committed:
Update 04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

Wheeeeeee
Explosives skill req changed to reflect current skill/lvl scaling

rosebrallier committed:
Update 07_Items.txt

So many capitalizations
Athletics skill replacing Str skill
Phrasing of Hacking checks

Turtlelord26 reviewed (approved):
Very shiny. Consistent templating = happy 🐢 

EDIT: There's another instance of the grenades requiring 10 skill here:
https://github.com/bleehu/Compound_X/blob/rosebrallier-skills-consistency-weapons/01_02_Combat_Rules.txt#L351
While you're in that file, would you please delete the bit right above this where it mentions dodging with Luck or Per? That's been moved to Reflex saves in the individual grenade descriptions. I wouldn't mind seeing the equivalent bit in the grenade launcher description also go away.
https://github.com/bleehu/Compound_X/pull/579/files#diff-8f4271a67677c39c3528636a3714eb99R242

rosebrallier committed:
Update 04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

rosebrallier committed:
Update 01_02_Combat_Rules.txt

Fixed some typography
Dropped grenade use pre-req to 6 Weapons - Explosive skill

Turtlelord26 commented:
:D 🐢 👍
